src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -191,7 +211,7 @@
             large
             outlined
             color="primary"
-            @click="isEditingType = false"
+            @click="resetType()"
           >
```

Comment:
This is to fix a bug in the Cancel button:
 it'a BC, 1 director -> enter the NR to change to a CC -> show error: less than 3 director
![image](https://github.com/user-attachments/assets/a2f9efc8-a86e-4ac2-96e6-1184c4789c9b)

click Cancel -> the name change and type change were incorrectly accepted:
![image](https://github.com/user-attachments/assets/5dd7b200-38b6-4456-9fc8-86a20f8ee9e5)

Reply:
Good catch.